holonote/annotate/display.py

```diff
@@ -251,7 +251,7 @@ def __init__(self, annotator: Annotator, **params) -> None:
 
         self._selection_info = {}
 
-        self._selection_enabled = True
+        self._selection_enabled = False
```

Comment:
To be consistent below:

```
            elif event.new == "+":
                self.annotator.editable_enabled = True
                self.annotator.selection_enabled = False
```

Reply:
Is this in this file? or is it in the `panel/app.py`?

Reply:
Why should both of them be True in either case?

Reply:
The annotator should be able to function without the widgets.